src/NuGet.Clients/NuGet.SolutionRestoreManager/Resources.resx

```diff
@@ -133,6 +133,10 @@
   <data name="NothingToRestore" xml:space="preserve">
     <value>All packages are already installed and there is nothing to restore.</value>
   </data>
+  <data name="NU1010" xml:space="preserve">
+    <value>Unable to read the project information: {0}</value>
```

Comment:
The error can be not because the project information failed to be read but maybe evaluated.  Maybe around the line "A failure was encountered while evaluating the project information: {0}" ?

Reply:
Only people with a reasonable MSBuild knowledge will understand what an evaluation is. To most .NET developers, particularly beginners, the more specific message may be difficult to understand.

What do you think about this?

```suggestion
    <value>Unable to parse the project information: {0}</value>
```

Reply:
Resolving, as I'm going to reuse NU1105 instead of creating a new code.